content/Watsonx/WatsonxAI/106.md

```diff
@@ -80,15 +80,30 @@ Before we can ingest data into Watson Discovery, we will start by creating a new
 
 In conjunction with Watson Discovey as our knowledge base, we will be making use of watsonx.ai foundation models to answer questions about the documents in our Watson Discovery Collection.
 
-After navigating to the watsonx home page [here](https://dataplatform.cloud.ibm.com/wx/home), we will want to open the Notebook  editor that we can use to run the notebook associated with this lab. If you don't know how to acccess watsonx.ai or are unsure how to get to the notebook editor, follow the link from the note below which will walk you through the process for accessing watsonx.ai as well as joining the notebook editor.
+After navigating to the watsonx home page [here](https://dataplatform.cloud.ibm.com/wx/home), we will want to open the Notebook editor that we can use to run the notebook associated with this lab. If you don't know how to acccess watsonx.ai or are unsure how to open to the notebook editor, follow the link from the note below which will walk you through the process for accessing watsonx.ai as well as joining the notebook editor.
 
-> A previous lab outlines how you can programmatically access watsonx.ai, you can follow part of that lab [here - Lab 102 Programmatic Access to watsonx.ai](/watsonx/watsonxai/102)
+### How to import a notebook into watsonx
 
-From that same Lab 102 in the note above, you can follow the same section for [importing Notebooks into watsonx](/watsonx/watsonxai/102#importing-a-notebook-into-watsonx)
+To import a notebook into watsonx follow the steps below:
 
-We will be importing the notebook for this lab using the following link for the **Notebook URL**:
+1. Login and navigate to the watsonx dataplatform main dashboard [here](https://dataplatform.cloud.ibm.com/wx/home)
 
- `https://raw.githubusercontent.com/ibm-build-lab/VAD-VAR-Workshop/main/content/WatsonX/WatsonxAI/files/rag-with-discovery.ipynb`
+2. Click the **Work with data and models in Python or R notebooks** card/tile
+
+    ![notebook_card](./images/106/10-Dashboard-Notebook-Tile.png)
+
+3. A new page with a modal menu titled **Work with data and models in Python or R notebooks** will appear. In this menu click the **URL** option in the left-hand navigation
+
+4. In the **Name** field type `{uniqueId}-rag-discovery`, where `uniqueId` represents your initials and the location of your workshop such as `ao-dallas-rag-discovery`; you can also add an optional description.
+
+5. For the **Select runtime** option use `Runtime 22.2 on Python 3.10 XXS`, larger runtimes use more tokens or cloud credits.
+
+6. In the **Notebook URL** section copy `https://raw.githubusercontent.com/ibm-build-lab/VAD-VAR-Workshop/main/content/WatsonX/WatsonxAI/files/rag-with-discovery.ipynb` into the field
```

Comment:
I think this URL needs to be: https://github.com/ibm-build-lab/VAD-VAR-Workshop/blob/main/content/Watsonx/WatsonxAI/files/rag-with-discovery.ipynb

Reply:
@krab7191 Ahhh.... good catch, there was a typo in the `raw.githubusercontent` url where its `WatsonX` instead of `Watsonx`. Let me know if works in the latest commit.